extensions/mssql/src/mssql.d.ts

```diff
@@ -724,13 +724,8 @@ declare module 'mssql' {
 	}
 
 	export const enum SystemDatabase {
-		master = 0,
-		msdb = 1
-	}
-
-	export const enum ProjectType {
```

Comment:
These were duplicates that didn't register because they were lowercased, I think.